CefSharp.BrowserSubprocess.Core/CefBrowserWrapper.h

```diff
@@ -26,15 +27,38 @@ namespace CefSharp
     {
     private:
         MCefRefPtr<CefBrowser> _cefBrowser;
+        // The entire set of possible JavaScript functions to
+        // call directly into.
+        JavascriptCallbackRegistry^ _callbackRegistry;
+        ConcurrentObjectRegistry<JavascriptAsyncMethodCallback^>^ _methodCallbacks;
     
     internal:
-        property JavascriptRootObjectWrapper^ JavascriptRootObjectWrapper;
+        property ConcurrentDictionary<int64, JavascriptRootObjectWrapper^>^ JavascriptRootObjectWrappers;
+        
+        property JavascriptCallbackRegistry^ CallbackRegistry
+        {
+            CefSharp::Internals::JavascriptCallbackRegistry^ get()
+            {
+                return _callbackRegistry;
+            }
+        }
+
+        property ConcurrentObjectRegistry<JavascriptAsyncMethodCallback^>^ MethodCallbackRegistry
```

Comment:
Should probably have `Async` in the name so it's clear which implementation it belongs to.